data/godzilliqa/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "GodZilliqa",
+    "symbol": "GODZ",
+    "email": "GodZilliqaDeFi@gmail.com",
+    "publicTeam": true,
```

Comment:
remove this

Reply:
Sorry my previous comment was referring to this line. But its listed as a valid field on the README.

Reply:
It is valid, but your project doesn't match the criteria for it, or the "holders" one for that matter

Reply:
Makes sense, we are going to link our public profiles soon enough. Thank you!